src/main/java/lotto/domain/DrawnNumbers.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.dto.LottoResultDto;
+
+/**
+ * 당첨 번호와 보너스 번호를 저장하는 클래스
+ */
+public class DrawnNumbers {
+    private final WinningNumbers winningNumbers;
+    private final BonusNumber bonusNumber;
+
+    private DrawnNumbers(final WinningNumbers winningNumbers, final BonusNumber bonusNumber) {
+        this.winningNumbers = winningNumbers;
+        this.bonusNumber = bonusNumber;
+    }
```

Comment:
```suggestion
    private DrawnNumbers(
        final WinningNumbers winningNumbers, 
        final BonusNumber bonusNumber
) {
        this.winningNumbers = winningNumbers;
        this.bonusNumber = bonusNumber;
    }
```

[개인적인 의견이에요]

자바/스프링 진영에서 생각보다 많이 적용되고 있는 개행 컨벤션이에요!
이렇다 할 규정은 없지만, 파라미터가 복수로 들어오면, 컴마 이후에 개행을 주어
파라미터별로 구분해서 쓰면, 훨씬 더 깔끔한 코드 작성이 가능합니다!

Reply:
자세한 설명과 코드 제안까지 직접 해주시다니 너무 감사드립니다 .. !!
코드 한 줄이 너무 길어 보기 힘들었는데, 저렇게 작성하면 더 깔끔하게 작성할 수 있겠네요. 😊😊